start.sh

```diff
@@ -103,7 +103,11 @@ if [ ! -x "$(command -v yq)"  ]; then
         exit 1
     elif [ "${OS}" == "linux" ]; then # if linux, assume it is canary, and install yq
         echo "Attempting to install yq"
-        wget https://github.com/mikefarah/yq/releases/download/v4.12.2/yq_linux_amd64 -O ~/bin/yq && chmod +x ~/bin/yq
+        wget https://github.com/mikefarah/yq/releases/download/v4.12.2/yq_linux_amd64 -O /usr/local/bin/yq && chmod +x /usr/local/bin/yq
```

Comment:
Looks good! 

I think we should run a sample run of this PR on the canaries though to ensure this is working as expected - 
IIRC, we had trouble using the `/usr/local/bin/yq` directory due to how the canary environments are configured. We ended up ensuring `yq` is in the same directory as `oc` when it is installed here - https://github.com/stolostron/canary-scripts/blob/2.4-bvt/install.sh#L32

Reply:
Ah, gotcha. I'll update to check for a valid `PATH`.

Reply:
Updated